UACloudLibraryServer/Controllers/InfoModelController.cs

```diff
@@ -104,13 +107,14 @@ public IActionResult GetAllNamesandIdentifiersAsync()
             [FromQuery][SwaggerParameter("Download NodeSet XML only, omitting metadata")] bool nodesetXMLOnly = false)
         {
             UANameSpace result = new UANameSpace();
-
-            result.Nodeset.NodesetXml = await _storage.DownloadFileAsync(identifier).ConfigureAwait(false);
-            if (string.IsNullOrEmpty(result.Nodeset.NodesetXml))
+            if (!omitXml)
```

Comment:
please change to nodesetXMLOnly

Reply:
omitXML has the opposite semantics: only return the UANameSpace meta data without the actual XML.

This is used for example in the REST fallback for UACloudLibClient.GetConvertedMetadataAsync. There is no other way to retrieve the namespace metadata efficiently.

Reply:
Got it. Then please name it metadataOnly.

Reply:
Yes, much better. Done.